tests/tasks/setup/eks/awscli-cp.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cluster-create
+  namespace: tekton-pipelines  
+spec:
+  description: |
+    Create an EKS cluster.
+    This Task can be used to create an EKS cluster for a given VPC Subnets, security groups and service role in an AWS account and write a kubeconfig to a desired location that
```

Comment:
Description is incorrect, `for a given VPC Subnets, security groups` task doesn't accepts these, it auto-discovers them.

Reply:
Yup, this has to be updated now that I added auto-discovery feature.